fesvr/htif.h

```diff
@@ -27,6 +27,7 @@ class htif_t : public chunked_memif_t
   bool done();
   int exit_code();
   void set_expected_xlen(unsigned int m) { expected_xlen = m; }
+  void set_load_offset(reg_t offset) { load_offset = offset; }
```

Comment:
The setter method seems like YAGNI to me, since we set it once to a constant.  Can we instead make `load_offset` a `const reg_t` that's initialized to `DRAM_BASE`?

Reply:
Yes of course, Idid this since I wasn't sure if htif would need to be independent from sim or not.